remote-helloworld-mdb/pom.xml

```diff
@@ -0,0 +1,365 @@
+                                </goals>
+                                <configuration>
+                                    <environmentVariables>
+                                        <JBOSS_HOME>${project.build.directory}/server</JBOSS_HOME>
+                                    </environmentVariables>
+                                    <includes>
+                                        <include>**/ProvisionedManagedMDBServletIT</include>
+                                    </includes>
+                                    <excludes>
+                                        <exclude>**/RemoteMDBServletIT</exclude>
+                                        <exclude>**/ManagedMDBServletIT</exclude>
+                                    </excludes>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>openshift</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.wildfly.plugins</groupId>
+                        <artifactId>wildfly-maven-plugin</artifactId>
+                        <version>${version.wildfly.maven.plugin}</version>
+                        <configuration>
+                            <feature-packs>
+                                <feature-pack>
+                                    <location>org.wildfly:wildfly-galleon-pack:${version.server}</location>
+                                </feature-pack>
+                                <feature-pack>
+                                    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:${version.cloud.fp}</location>
+                                </feature-pack>
+                            </feature-packs>
+                            <layers>
+                                <layer>cloud-server</layer>
+                                <layer>ejb</layer>
+                                <layer>remote-activemq</layer>
+                            </layers>
+                            <filename>ROOT.war</filename>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>package</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <!-- An optional Arquillian testing profile that executes tests in your JBoss EAP instance.
+            This profile will start a new JBoss EAP instance, and execute the test, shutting it down when done.
+            Run with: mvn clean verify -Parq-managed -->
+            <id>arq-managed</id>

```

Comment:
This is a more common question thought, but:

Do we still need this profile? AFAIK the only profiles that are mentioned in Readme are:
* arq-remote
* provisioned-server
* openshift

cc @emmartins